app/controllers/application_controller.rb

```diff
@@ -230,5 +231,10 @@ def user_strategies(user_id)
   def user_moments(user_id)
     Moment.where(user_id: user_id)
   end
+
+  def set_raven_context
+    Raven.user_context(id: current_user.id)
+    Raven.extra_context(params: params.to_unsafe_h, url: request.url)
```

Comment:
Using to_unsafe_h was suggested in the Sentry official setup docs lol.

Reply:
Hmm I don't know if we should expose email and id together in the error logs. Even though our app isn't officially HIPAA compliant, I don't want potentially sensitive user information to show up in the logs that is identifiable. But then again, only specific/requested contributors in our org would have access... 🤔